IAMOTZ/ievents
# #153415589 Implement the all centers page with react

#### What does this PR do?
It uses react and redux to implement the page that shows all the centers in the appllication
#### Description of Task to be completed?
User should be able to get all centers
#### How should this be manually tested?
- clone the repo
- cd into the project directory
- Run `npm install` to install all the dependency
- Run `npm run react:dev`
- visit the port where the webpack-dev-server started
- try posting some centers through the API endpoint
- navigate to the centers page, all the centers should be laid out in cards
#### Any background context you want to provide?
- The frontend still uses the locally hosted server, so you would have to create a .env file to fulfill all the needed environment variables, the run `npm run server:dev`
#### What are the relevant pivotal tracker stories?
#153415589 
#### Screenshots (if appropriate)
![screencapture-localhost-8080-centers1-1512564643842](https://user-images.githubusercontent.com/26048536/33662571-76dc68da-da8c-11e7-9d16-67822f837c9f.png)
#### Questions:
None
